src/client/app/components/calculator.jsx

```diff
@@ -1,19 +1,55 @@
 /* eslint-disable no-trailing-spaces */
 import React from "react";
+import PropTypes from "prop-types";
+import Card from "@material-ui/core/Card";
+import { withStyles } from "@material-ui/core/styles";
 import PanelIO from "./panelio.jsx";
 import NumPad from "./numpad.jsx";
 
+const cardStyles = {
+    card: {
+        display: "block",
+        minWidth: 250,
+        backgroundColor: "grey",
+        margin: "auto"
+    },
+    cardParent: {
+        position: "absolute",
+        left: 0,
+        to: 0,
+        height: "100%",
+        width: "100%",
+        display: "flex",
+        justifyContent: "center",
+        alignItems: "center"
+    }
+    /*
+    bullet: {
+        display: "inline-block",
+        margin: "0 2px",
+        transform: "scale(0.8)"
+    },
+    title: {
+        fontSize: 14
+    },
+    pos: {
+        marginBottom: 12
+    } */
+};
+
 class Calculator extends React.Component {
 
     state={
         number: 0
     };
+
     changeNumber = (value) => this.setState({number: value});
 
     render () {
+        const { classes } = this.props;
         return (
-            <div style={styles.calcContainer}>
-                <div>Works only with . when decimal matters</div>
+            <div className={classes.cardParent}>
```

Comment:
@myxomor Very bad className. 

read this https://ru.bem.info/methodology/html/


![image](https://user-images.githubusercontent.com/10912312/48960746-668a6b80-ef80-11e8-9c97-020051ea24be.png)